### PR TITLE
[TASK] Rework chapters on reviewing and contributing patches and issues

### PR DESCRIPTION
**Pull request by garvinhicking**

### CODE REVIEW

File: Documentation/Setup/UseStyleguide.rst [hunk 1]
```diff
@@ -9,11 +9,11 @@
 Use EXT:styleguide
 ==================
 
-It is recommended to use the styleguide extension to auto generate several
+It is recommended to use the `styleguide`` extension to auto generate several
```
Comment (garvinhicking) [inline Documentation/Setup/UseStyleguide.rst:12]:
```suggestion
It is recommended to use the `styleguide` extension to auto generate several
```

**Comment by linawolf** [comment]
Does this patch need the review of a Core merger :tm:?

**Review by sbuerk — DISMISSED** [review]

File: Documentation/HandlingAPatch/FindAReview.rst [hunk 2]
```diff
@@ -91,6 +91,29 @@ Gerrit: Search
...
 
+GitHub: Links to Gerrit
+=======================
+
+Each commit to the TYPO3 Git repository comes with some metadata
+linking issue and patch reviews, for example:
+
+..  code-block::
+
+   [TASK] Provide PSR-7 Request in PolicyMutatedEvent
```
Comment (linawolf) [inline Documentation/HandlingAPatch/FindAReview.rst:102]:
```suggestion
   [TASK] Provide PSR-7 Request in PolicyMutatedEvent
   
```

**Comment by garvinhicking** [comment]
> Does this patch need the review of a Core merger ™️?

I would like this very much and I'll go fishing for one, unless @sbuerk give's his seal of approval maybe :-D

File: Documentation/HandlingAPatch/Review.rst [hunk 3]
```diff
@@ -194,7 +204,26 @@ be solved until this or that is fixed". Some hints on using -1 in reviews:
    a deeper knowledge of this subsystem, to take a look at it. I do not want
    this patch to be merged until this is sorted out and will vote -1 for now
    for this reason."
+*  If you are on the "receiving end" of a "-1" or even a "-2" vote, please do not
```
Comment (froemken) [inline Documentation/HandlingAPatch/Review.rst:207]: Well spoken...aahh...written ;-)

File: Documentation/Troubleshooting/Index.rst [hunk 4]
```diff
@@ -250,39 +250,56 @@ A valid private key in OpenSSH format starts with the following lines:
...
+If you cherry pick a patch for review, you might encounter a merge conflict with a generated asset file
+(JavaScript or CSS build files):
 
 .. code-block:: none
    :caption: result
 
    Mergeconflict in typo3/sysext/backend/Resources/Public/Css/backend.css
 
-To resolve the conflict, use the following workflow:
+To resolve the conflict, do not try to resolve conflichts in a huge 1-line file, but instead
```
Comment (froemken) [inline Documentation/Troubleshooting/Index.rst:281]:
```suggestion
To resolve the conflict, do not try to resolve conflicts in a huge 1-line file, but instead
```

**Review by froemken — APPROVED** [review]
Holy...what have you done? Awesome. You have written a book :D  Nice update for the Coding Night at DevDays, right?

**Review by linawolf — APPROVED** [review]

**Comment by garvinhicking** [comment]
> Holy...what have you done? Awesome. You have written a book :D Nice update for the Coding Night at DevDays, right?

Thank you... indeed it felt as if I did. I hope it's of worth somewhere along the way...

File: Documentation/HandlingAPatch/ResolveMergeConflicts.rst [hunk 5]
```diff
@@ -229,6 +230,33 @@ The merge conflict will show:
...
 There may be more than one conflict in a file!
 
+Special case: JavaScript/CSS merge conflicts
+--------------------------------------------
+
+JavaScript and CSS assets are build from sources in the monorepo, with the commands:
+
+.. code-block:: bash
+
+   Buid/Scripts/runTests.sh -s buildCss
```
Comment (georgringer) [inline Documentation/HandlingAPatch/ResolveMergeConflicts.rst:240]: should we add -s clean before

File: Documentation/ReportingAnIssue/Index.rst [hunk 6]
```diff
@@ -205,8 +224,12 @@ TYPO3 version
 
 (for bugs only)
 
-Choose the TYPO3 version, where the error occurs.
+Choose the TYPO3 version, where the error occurs. Ideally try
+to report an issue for the latest TYPO3 version where your issue
+applies to.
 
+Older TYPO3 versions outside of the LTS support range are less
```
Comment (georgringer) [inline Documentation/ReportingAnIssue/Index.rst:231]: outside of LTS receive no public support but only via ELTS and it is up to the ELTS team (maybe ask one of the gmbh people for details) what is fixed and what not.
for active versions, see https://get.typo3.org/ > old stable versions like current 11.5 also only get "important" or "relevant" backports.

**Comment by linawolf** [comment]
@georgringer thanks for your feedback, this PR is already merged. Please open an issue or provide a follow up PR

**Comment by garvinhicking** [comment]
> @georgringer thanks for your feedback, this PR is already merged. Please open an issue or provide a follow up PR

I will address this, thanks to both of you

**Pull request merged**